apps/arweave/src/ar_tx_emitter.erl

```diff
@@ -42,26 +42,42 @@ start_link(Name, Workers) ->
 
 init(Workers) ->
 	gen_server:cast(?MODULE, process_chunk),
-	{ok, #state{ workers = queue:from_list(Workers), currently_emitting = sets:new() }}.
+	State = #state{ workers = queue:from_list(Workers)
+		      , currently_emitting = sets:new()
+		      },
+	{ok, State}.
 
 handle_call(Request, _From, State) ->
 	?LOG_WARNING([{event, unhandled_call}, {module, ?MODULE}, {request, Request}]),
 	{reply, ok, State}.
 
 handle_cast(process_chunk, State) ->
 	#state{ workers = Q, currently_emitting = Emitting } = State,
+
+	% only current (active) peers should be used, using lifetime
+	% peers will create unecessary timeouts. The first to
+	% contact are the trusted peers
 	TrustedPeers = ar_peers:get_trusted_peers(),
-	Peers = (ar_peers:get_peers(lifetime) -- TrustedPeers) ++ TrustedPeers,
-	{ok, Config} = application:get_env(arweave, config),
-	{Q2, Emitting2} = emit(ar_mempool:get_propagation_queue(), Q, Emitting, Peers,
-			Config#config.max_propagation_peers, ?CHUNK_SIZE),
+	CurrentPeers = ar_peers:get_peers(current) -- TrustedPeers,
```

Comment:
Everything other than this change is refactoring, right? i.e. we shouldn't expect any functional change other than this line?